static/app/views/starfish/table.tsx

```diff
@@ -198,7 +200,8 @@ class _Table extends Component<Props, State> {
             onClick={this.handleSummaryClick}
             style={{display: `block`, width: `100%`}}
           >
-            {rendered}
+            {prefix}
+            {dataRow.transaction}
```

Comment:
This is janky, but since we won't be keeping this module around longterm i dont want to update the discover field renderers for this